testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java

```diff
@@ -55,24 +53,17 @@ public static boolean isPrestoContainer(String name)
     public static void configureTempto(Environment.Builder builder, ResourceProvider configDir)
     {
         builder.configureContainer(TESTS, dockerContainer -> {
-            Path path = configDir.getPath("tempto-configuration.yaml");
-            String suffix = getParentDirectoryName(path) + "-" + randomSuffix();
+            String suffix = builder.getEnvironmentName() + "-" + randomSuffix();
```

Comment:
So it will be fixed. As it will be always called `TESTS`. Am I right?

Reply:
No. The environmentName here is the PT env name. So it will be things like `singlenode-kerberos-hive-impersonation` or `multinode-tls-kerberos` etc.

I think you are confusing it with container name.

Reply:
does `builder.getEnvironmentName()` buy us anything compared to random suffix alone?

or can we name these files like `gizmo-1`, `gizmo-2`?

Reply:
Good point. The motivation behind current approach was to identify what file was loaded. If we load several tempo configs from different env extenders then their names won't be any better than using gizmo. We know what is the env name already.

Reply:
Good point. Do you think we should just use the `randomSuffix` part and drop the rest?

Reply:
Maybe we can leave some comment. It was not that obvious.

Reply:
I'll add a comment - good point. Would prevent me from opening another PR in the future. 😄